scripts/release/build_all.sh

```diff
@@ -29,16 +29,11 @@ if [[ ! -d "./icicle" || ! -d "./scripts" ]]; then
   exit 1
 fi
 
-# Build Docker images
-echo "Building Docker images..."
-# Ubuntu 22.04, CUDA 12.2.2
-docker build -t icicle-release-ubuntu22-cuda122 -f ./scripts/release/Dockerfile.ubuntu22 .
-# Ubuntu 20.04, CUDA 12.2.2
-docker build -t icicle-release-ubuntu20-cuda122 -f ./scripts/release/Dockerfile.ubuntu20 .
-# ubi8 (rhel compatible), CUDA 12.2.2
-docker build -t icicle-release-ubi8-cuda122 -f ./scripts/release/Dockerfile.ubi8 .
-# ubi9 (rhel compatible), CUDA 12.2.2
-docker build -t icicle-release-ubi9-cuda122 -f ./scripts/release/Dockerfile.ubi9 .
+# Download latest build images
```

Comment:
I assume you built and pushed them manually? can you commit a script for doing so as well? so we can easily build and push when updating the dockers

Reply:
@yshekel There is a github workflow for it https://github.com/ingonyama-zk/icicle/blob/13220b56671f9d48caeac1c60ccff5e9dc9a0497/.github/workflows/build-release-docker-img.yml.

I can make it a script if you want so its possible to run outside of github actions